### ISSUE TITLE
Version conflict of RTools (v.4.3) and R (v.4.4)

### ISSUE THREAD

**Issue by hkim-fda** [dongyuanwu/RSBID#1]
Hello.  `RSBID` used to work with R-v.4.3 and RTools v. 4.3 on Windows 10 without problem of its installation, but I have got the following errors after upgrading to R- v.4.4.  The thing is that `install_github()` directs to start installing the older RTools (v.4.2)  leading to failure because the user privilege issue (not an administrator) since I am using the company-assigned computer that restricts all software installation unless I submit a ticket for a request to IT-department.  Strangely,  the computer has already the higher version of RTools (v.4.3) than the RSBID tries to install.  I do not understand why your package is not working with RTools v.4.3 and R v.4.4.

```
 library(devtools)
Loading required package: usethis
> install_github("dongyuanwu/RSBID")
Downloading GitHub repo dongyuanwu/RSBID@HEAD
trying URL 'https://cran.rstudio.com/bin/windows/Rtools/rtools42/files/rtools42-5355-5357.exe'
Content type 'application/x-msdownload' length 482123824 bytes (459.8 MB)
downloaded 459.8 MB

Error: Failed to install 'RSBID' from GitHub:
  Could not find tools necessary to compile a package
Call `pkgbuild::check_build_tools(debug = TRUE)` to diagnose the problem.
> pkgbuild::check_build_tools(debug = TRUE)
trying URL 'https://cran.rstudio.com/bin/windows/Rtools/rtools42/files/rtools42-5355-5357.exe'
Content type 'application/x-msdownload' length 482123824 bytes (459.8 MB)
downloaded 459.8 MB

Error: Could not find tools necessary to compile a package
Call `pkgbuild::check_build_tools(debug = TRUE)` to diagnose the problem.
```

I wonder if you can update the package for working on Windows.

Thanks.

**Comment by dongyuanwu** [comment]
Hi, thanks for using `RSBID`! I tried rebuilding the package and it seems that the problem doesn't originate from the code. Below is the possible solution I found:

I tried to upgrade to R v4.4 and encountered the same issue
```
Could not find tools necessary to compile a package.
```
According to the page https://cran.r-project.org/bin/windows/Rtools/, R v4.4 requires the use of Rtools v4.4 instead of Rtools v4.3. Subsequently, I manually installed Rtools v4.4 from that page, and everything worked well.

I guess your computer does not have the correct version of Rtools for R v4.4 so `install_github()` attempted to install something (though I'm not sure why v4.2...).

Feel free to let me know if you still have the issue after installing Rtools v4.4. :)
